grtc/crypto.js

```diff
@@ -0,0 +1,32 @@
+const forge = require( 'node-forge' );
```

Comment:
Crypto packages tend to be extremely large for browser consumption. This one is approximately [266kb minified and gzipped](https://unpkg.com/node-forge@0.7.0/dist/forge.min.js). I'd sought similar in https://github.com/Automattic/wp-calypso/pull/17356, though I'm unsure of a good alternative for RSA key generation as you need here. Do you know of other options we could evaluate? I think the size as-is is quite excessive.

Reply:
We can remove the support of encryption and trust the turn servers in the relay. It can be removed completely.

Reply:
Removed completely with https://github.com/WordPress/gutenberg/pull/1877/commits/4ebad1270a9b2a914cf3544cdb7e7c3559686494.